StandOutLib/build.gradle

```diff
@@ -0,0 +1,33 @@
+apply plugin: 'com.android.library'
```

Comment:
I see you've renamed library/ to StandOutLib/ 

please rename back to library/ to keep the directory structure as before